### ISSUE TITLE
NYT not working on Chrome canary 77.0.3827.0

### ISSUE THREAD

**Issue by This-will-be-your-username** [iamadamdev/bypass-paywalls-chrome#235]
NYT works properly on 75.0.3770.90, at least with uBlock Origin also installed, however on Chrome 77.0.3827.0 (and previous couple versions, sorry, don't have the past version numbers) this results in the "You’re in private mode." pop-up.

**Comment by This-will-be-your-username** [comment]
Still broken on 77.0.3828.0

**Comment by iamadamdev** [comment]
As long as you have uBlock Origin installed you shouldn't really need my plugin for NYT..

**Comment by This-will-be-your-username** [comment]
Looks like I didn’t test this well enough. After removing my profile to start from scratch and installing both uBlock Origin
https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm

And uBlock Origin Extra
https://chrome.google.com/webstore/detail/ublock-origin-extra/pgdnlhfefecpicbbihgmbmffkjpaplco

The paywall popup isn’t displaying anymore (without bypass-paywalls installed). I don’t know if there was something in my profile that was doing it (as I had cleared all site data for all time), another extension interfering that the profile wipe resolved, or if it’s the uBlock Origin Extra extension that’s got this working again, but a couple more troubleshooting steps here for others to try if they find themselves with the same issue.

Closing issue.
